src/services/bundle.service.js

```diff
@@ -11,10 +11,10 @@ const logger = loggers.get('default');
 
 /**
  * Creates transaction-response Bundle
- * @param {*} results - request results
- * @param {*} res - an object containing the response
- * @param {*} type - bundle type
- * @param { boolean } xprovenanceIncluded - X-Provenance header was included and
+ * @param {Array} results - an array of equest result objects
```

Comment:
```suggestion
 * @param {Array} results - an array of request result objects
```

Reply:
I also noticed the `-` in the jsdoc comments here. Not in scope, but do we want to establish an official style for this?

Reply:
I like the dashes!